src/widgets/numeric-selector/numeric-selector.js

```diff
@@ -68,6 +68,7 @@ const usage = `Usage: numericSelector({
  *
  * The values inside this attribute must be JavaScript numbers and not strings.
  * @type {WidgetFactory}
+ * @devNovel NumericSelector
```

Comment:
I wonder if we can't edit the dev novel to have the same name as the widget, so there doesn't need to be duplicate config 🤔

Reply:
That's a good thought. Why did you go for this approach @samouss ?

Reply:
I thought it was a bit more flexible and less magical to have the name in the file. Note that the annotation is required in order to generate the "Showcase" link because in some cases the widget don't have a story (e.g. `Configure`).

Reply:
What about leaving the default to "there's a story", making sure the name in the novel is the same as the widget name, and adding something like `@suppressDevNovel` to those that specifically don't have a novel?

Reply:
I prefer to have the opt-in in a "positive" way rather than the other. It feels more natural to add them rather than suppress them.

Reply:
okay, just suggesting it, since I thought it might be easy to forget about it making a new widget, but it's fine like this.

Reply:
I like the opt-in aspect. Sounds good to me.